integrationtests/pom.xml

```diff
@@ -33,58 +38,31 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-slf4j-impl</artifactId>
+            <groupId>io.kroxylicious.testing</groupId>
+            <artifactId>testing-junit5-extension</artifactId>
+            <version>${kroxy.extension.version}</version>
```

Comment:
Remove the version tag from this declaration and add it to the dependencyManagement element of the parent pom.

Reply:
Sorry @k-wall very long time without working with pom files. I think it is ok now

Reply:
agreed, looks good